qutebrowser/browser/hints.py

```diff
@@ -1023,6 +1024,20 @@ def hint_follow(self, select: bool = False, keystring: str = None) -> None:
         else:
             self._fire(keystring)
 
+    @cmdutils.register(instance='hintmanager', scope='window',
+                       modes=[usertypes.KeyMode.hint])
+    def hints_cycle(self, reverse: bool = False) -> None:
+        """Bring hints covered by other hints to the foreground.
+
+        Args:
+            reverse: Cycle in reverse direction.
+        """
+        hintcontext = self._context
```

Comment:
I don't think this is needed FWIW.

Reply:
You don't think which is needed?

Reply:
A `hintcontext` variable, rather than using `self._context` directly. It seems like having two names for the same thing without a benefit (other than perhaps saving two chars).

Reply:
Oh yeah, it's probably an artifact of a previous version or wrestling with mypy.